src/tpl/form-component.tpl.html

```diff
@@ -206,7 +206,8 @@
 	        <span ng-if="!config.autocompleteConfig.displayKey" md-highlight-text="config.autocompleteConfig.searchText" md-highlight-flags="^i" title="{{item}}"  data-auto="{{config.code}}-{{item}}-autocomplete">{{ item }}</span>
 	      </md-item-template>
 	      <md-not-found>
-	        Nothing results matching "{{config.autocompleteConfig.searchText}}" were found.
+					<span ng-if="config.autocompleteConfig.searchText.length === 0">No options available</span>
+					<span ng-if="config.autocompleteConfig.searchText.length > 0">Nothing results matching "{{config.autocompleteConfig.searchText}}" were found.</span>
```

Comment:
correct type from `Nothing results matching "{{config.autocompleteConfig.searchText}}" were found.` to `No results matching "{{config.autocompleteConfig.searchText}}" were found.`

Reply:
Done